scripts/scanEventsFromUncover.js

```diff
@@ -0,0 +1,131 @@
+        logger.info(`newValidatorSetId: ${newValidatorSetId}`);
+        logger.info(`event proof::${JSON.stringify(eventProof)}`);
+        const proof = {
+            eventId: eventProof.eventId,
+            validatorSetId: eventProof.validatorSetId,
+            r: eventProof.r,
+            s: eventProof.s,
+            v: eventProof.v
+        };
+        try {
+            const gasEstimated = await bridge.estimateGas.setValidators(newValidators, newValidatorSetId, proof, {gasLimit: 500000});
+
+            logger.info(JSON.stringify(await bridge.setValidators(newValidators, newValidatorSetId, proof, {gasLimit: gasEstimated.add(BUFFER)})));
+            await updateLastEventProcessed(eventId, blockHash.toString());
+            const balance = await ethers.provider.getBalance(txExecutor.address);
+            logger.info(`Balance is: ${balance}`);
+            const gasPrice = await ethers.provider.getGasPrice();
+            logger.info(`Gas price: ${gasPrice.toString()}`);
+            const gasRequired = gasEstimated.mul(gasPrice);
+            logger.info(`Gas required: ${gasRequired.toString()}`);
+            if (balance.lt(gasRequired.mul(2))) {
+                const {statusCode, data} = await curly.post(`https://hooks.slack.com/services/${process.env.SLACK_SECRET}`, {
+                    postFields: JSON.stringify({
+                        "text": ` 🚨 To keep the validator relayer running, topup the eth account ${txExecutor.address} on CENNZnets ${process.env.NETWORK} chain`
+                    }),
+                    httpHeader: [
+                        'Content-Type: application/json',
+                        'Accept: application/json'
+                    ],
+                });
+                logger.info(`Slack notification sent ${data} and status code ${statusCode}`);
+            }
+        } catch (e) {
+            logger.warn('Something went wrong:');
+            logger.error(`Error: ${e}`);
+        }
+    } else if (!eventProof){
+        logger.info(`Could not retrieve event proof for event id ${eventId} from derived
+        query api.derive.ethBridge.eventProof at ${timeoutMs} timeout`);
+    }
+}
+
+async function main (networkName, bridgeContractAddress) {
+    networkName = networkName || 'local';
+
+    const connectionStr = process.env.MONGO_URI;
+    await mongoose.connect(connectionStr);
+
+    const api = await Api.create({network: networkName});
+    logger.info(`Connect to cennznet network ${networkName}`);
+    [txExecutor] = await ethers.getSigners();
+
+    // Get the bridge instance that was deployed
+    const Bridge = await ethers.getContractFactory('CENNZnetBridge');
+    logger.info('Connecting to CENNZnet bridge contract...');
+    const bridge = await Bridge.attach(bridgeContractAddress);
+    await bridge.deployed();
+    logger.info(`CENNZnet bridge deployed to: ${bridge.address}`);
+    logger.info(`Executor: ${txExecutor.address}`);
+
+
+    const response = await axios.get(
```

Comment:
everything remains same, except for the events here are fetched from uncover api and are then submitted